packages/atomic/cypress/integration/no-results-v1.cypress.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  buildTestUrl,
+  injectComponent,
+  setUpPage,
+} from '../utils/setupComponent';
+import {generateAliasForSearchBox} from './search-box-selectors';
+
+describe('No Results Test Suites', () => {
+  const tag = 'atomic-no-results-v1';
+  const component = (attributes = '') => `<${tag} ${attributes}></${tag}>`;
+  const searchBox = '<atomic-search-box></atomic-search-box>';
+  const wait = 1000;
+
+  it('should not be visible when there are results', () => {
+    setUpPage(component());
+    cy.get(tag).should('not.be.visible');
+  });
+
+  it('should be visible when there are no results', () => {
+    cy.visit(buildTestUrl('q=gahaiusdhgaiuewjfsf'));
+    injectComponent(component() + searchBox);
+    cy.wait(wait);
```

Comment:
The hardcoded wait should not be necessary, we usually wait for API calls to complete, there are examples in the facets v1 for this too